src/core/include/iDynTree/Core/VectorFixSize.h

```diff
@@ -277,37 +259,17 @@ namespace iDynTree
     template<unsigned int VecSize>
     VectorFixSize<VecSize> & VectorFixSize<VecSize>::operator=(const Span<const double>& vec) {
         assert(VecSize == vec.size());
-        std::memcpy(this->m_data, vec.data(), VecSize*sizeof(double));
+        std::memcpy(this->data(), vec.data(), VecSize * sizeof(double));
```

Comment:
Why not using the Eigen assignement here?

Reply:
Sorry I did not notice that the input was a span at first. Nevertheless, you can do something like
```	  
Eigen::Map< const Eigen::Matrix<double, VecSize, 1>> map(vec.data(), vec.size());
 Eigen::Matrix<double, VecSize, 1>::operator=(map);
 return *this;
```
I would avoid using ``memcpy`` since in principle ``Eigen`` may handle its memory differently.

Reply:
Done in https://github.com/robotology/idyntree/pull/709/commits/8fc84cd09206b04f8a69ef4fc859425424608069